x/dispensation/types/types.go

```diff
@@ -6,20 +6,20 @@ package types
 func NewQueryAllDistributionsResponse(distributions Distributions, height int64) QueryAllDistributionsResponse {
 	return QueryAllDistributionsResponse{
 		Distributions: distributions.Distributions,
-		Height: height,
+		Height:        height,
 	}
 }
 
 func NewQueryRecordsByDistributionNameResponse(distributionRecords DistributionRecords, height int64) QueryRecordsByDistributionNameResponse {
 	return QueryRecordsByDistributionNameResponse{
 		DistributionRecords: &distributionRecords,
-		Height: height,
+		Height:              height,
```

Comment:
unnecessary indent

Reply:
@pgoos this is an auto formatting mechanism that is part of the analysis tool in the golang toolset.